core/src/main/java/apoc/cypher/CypherInitializer.java

```diff
@@ -37,32 +39,32 @@ public boolean isFinished() {
         return finished;
     }
 
+    public GraphDatabaseAPI getDb() {
+        return db;
+    }
+
     @Override
     public void available() {
 
         // run initializers in a new thread
         // we need to wait until apoc procs are registered
         // unfortunately an AvailabilityListener is triggered before that
         Util.newDaemonThread(() -> {
-
             try {
-                awaitApocProceduresRegistered();
+                final boolean isSystemDatabase = db.databaseName().equals(GraphDatabaseSettings.SYSTEM_DATABASE_NAME);
+                if (!isSystemDatabase) {
```

Comment:
but if `isSystemDatabase` is true none of the code is executed, not the await for apoc nor collecting the config (it will return an empty list)

so we can also just return at the beginning of `available()` for the systemdb